profile/profile.md

```diff
@@ -0,0 +1,64 @@
+
+**Competent:** JavaScript, HTML/CSS, Python, Docker, Algorithms/Data structures, Jupyter notebook, Pandas
+
+**Familiar:** C#, Vue, Android, React, Angular
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_YerinAn](https://github.com/jarviscanada/jarvis_data_eng_YerinAn)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/linux_sql)]: Developed an internal tool to collect and save both server hardware specifications and on-going Linux resource usage data so that users can know their cluster status better. ⋅	Utilized the docker container to provision the database ⋅	Created bash scripts for monitoring nodes ⋅	Developed SQL queries to store and retrieve data
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/core_java)]:
+      
+  - Twitter App: In development
+  - JDBC App: Implemented an application that uses JDBC to build a connection between Java and PostgreSQL database. developed CRUD operations by using the DAO design pattern.
+  - Grep App: Developed an application that has the functionality to a simplified the Linux grep command using Java. Used the lambda function and stream API from Java to optimize the final project performance.   
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/springboot)]: In development
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/python_data_anlytics)]: In development
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/hadoop)]: In development
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/spark)]: In development
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_YerinAn/tree/master/cloud_devops)]: In development
+
+
+## Highlighted Projects
+**MODERN ERP application** [[GitHub](https://github.com/caiquebispoferreira/resourceserver)]: Created an application that any company or customer can make their online website based on their products and analyze the online business. Worked with Java 11, Vue, Javascript, Spring boot, and MS SQL.
+
+**WebGaming - Space defenders** [[GitHub](https://github.com/Calvin05/WebGamming-Group1)]: Designed a web application which is a space shooting game that uses Typescript, Javascript and CreateJs, and it works with a keyboard. Adjusted three stages that the difficulty level through different enemies, options, items and a high scoreboard.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**: Developed Data Engineering projects in Java using technologies such as Git, Java, Maven, Docker, Springboot, RDBMS/SQL, Python. Collaborated with a team in daily Scrum meetings to implement Agile development practices.
```

Comment:
too short. talk more about tech skills and soft skills (collaboration, agile/scrum, communication, leadership, etc..)